src/include/liburing.h

```diff
@@ -226,6 +226,7 @@ int io_uring_register_iowq_max_workers(struct io_uring *ring,
 				       unsigned int *values);
 int io_uring_register_ring_fd(struct io_uring *ring);
 int io_uring_unregister_ring_fd(struct io_uring *ring);
+int io_uring_close_ring_fd(struct io_uring *ring);
```

Comment:
If you export a function, you should also put it in the `liburing.map` as well.

Reply:
@ammarfaizi2 I hadn't realized that that was updated commit-by-commit rather than just prior to release. Will fix.

Reply:
Fixed.